apps/ledger-live-desktop/src/renderer/families/ethereum/EditTransactionModal/steps/StepSummaryFooter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from "react";
+import { Trans } from "react-i18next";
+import { getMainAccount } from "@ledgerhq/live-common/account/index";
+import Button from "~/renderer/components/Button";
+import { StepProps } from "../types";
+import { apiForCurrency } from "@ledgerhq/live-common/families/ethereum/api/index";
+import ErrorBanner from "~/renderer/components/ErrorBanner";
+import { TransactionHasBeenValidatedError } from "@ledgerhq/errors";
+
+export class StepSummaryFooter extends PureComponent<StepProps> {
+  state = {
+    transactionHasBeenValidated: false,
+  };
+
+  onNext = async () => {
+    const { transitionTo } = this.props;
+    transitionTo("device");
+  };
+
+  componentDidMount() {
+    const { account, parentAccount, transactionHash } = this.props;
+    if (!account || !transactionHash) return;
+    const mainAccount = getMainAccount(account, parentAccount);
+    if (mainAccount.currency.family !== "ethereum") return;
+    apiForCurrency(mainAccount.currency)
+      .getTransactionByHash(transactionHash)
+      .then((tx: { confirmations?: number }) => {
+        if (tx.confirmations) {
+          this.setState({ transactionHasBeenValidated: true });
+        }
+      });
+  }
+
+  render() {
+    const { account, status, bridgePending } = this.props;
+    if (!account) return null;
+    const { errors } = status;
+    // exclude "NotOwnedNft" and "NotEnoughNftOwned" error if it's a nft speedup operation
+    let errorCount = Object.keys(errors).length;
+    if (
+      errors.amount &&
+      (errors.amount.name.includes("NotOwnedNft") ||
+        errors.amount.name.includes("NotEnoughNftOwned"))
```

Comment:
Same thing here, either use the "helper" or favor using `instanceof` rather than erro names

Reply:
fixed